prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/TranslationFunctions.kt

```diff
@@ -0,0 +1,68 @@
+package gov.cdc.prime.router.fhirengine.translation.hl7.utils
+
+import ca.uhn.fhir.model.api.TemporalPrecisionEnum
+import ca.uhn.hl7v2.model.v251.datatype.DTM
+import org.hl7.fhir.r4.model.BaseDateTimeType
+
+/**
+ * This interface contains the required method signatures required to implement FHIR transloation
+ * functions. It contains the default dateTime convertion function.
+ */
+interface TranslationFunctions {
+
+    /**
+     * Convert a FHIR [dateTime] to the format required by HL7
+     * @return the converted HL7 DTM
+     */
+    fun convertDateTimeToHL7(dateTime: BaseDateTimeType, appContext: CustomContext? = null): String {
```

Comment:
Why is the implementation of this function in the interface rather than the implementation class?

Reply:
It is used for the default function.